TEST.md

```diff
@@ -2,7 +2,7 @@
 
 Some test behaviors are controlled by test overrides set via environment variables.
 They only work when the CMake Option `SFS_ENABLE_TEST_OVERRIDES` is set to `ON`.
-To enable that during build, you must use the switch `-EnableTestOverrides` (`--enable-test-overrides` on Linux) with the build scripts. Click [here](README.md#building) for more. See also [here](README.md#building-with-cmake-vscode-extension) for how to set when using VSCode.
+To enable that during build, you must use the switch `-EnableTestOverrides` (`--enable-test-overrides` on Linux) with the build scripts. See [building](README.md#building) for more information. See also [building with cmake vscode extension](README.md#building-with-cmake-vscode-extension) for how to set when using VSCode.

```

Comment:
This is based on an accessibility complaint (for people using screen readers):

#### Do not use `(click) here` links

For more information, see:
* https://www.w3.org/QA/Tips/noClickHere
* https://webaim.org/techniques/hypertext/link_text
* https://granicus.com/blog/why-click-here-links-are-bad/
* https://heyoka.medium.com/dont-use-click-here-f32f445d1021
```
(?i)(?:>|\[)(?:(?:click |)here|link|(?:read |)more)(?:</|\]\()
```

Reply:
Cool!